flight/Libraries/math/smoothcontrol.c

```diff
@@ -0,0 +1,217 @@
+#include <stdio.h>
+#include <stdint.h>
+#include <math.h>
+
+#include "openpilot.h"
+#include "pios.h"
+#include "smoothcontrol.h"
+
+#define ASSERT(x) if(x) PIOS_Assert(0);
+#define MAX(v,x) ((v) > (x) ? (x) : (v))
```

Comment:
We do already have higher quality ones: https://github.com/d-ronin/dRonin/blob/ebb1954b074bc168491bf11bdb639ddced14a0f9/flight/Libraries/math/misc_math.h#L38